frontend/src/components/pages/UserActions/ActivityDashBoardBarGraph/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from "react";
+import Presentation from "./Presentation";
+
+import { connect } from "react-redux";
+
+import { eventTypeColors } from "../../../../colors";
+import { prepareDataForBarGraph } from "./utils";
+
+export function ActivityDashboardBarGraphUnconnected({
+  eventTypes,
+  activityLogDayCounts,
+}) {
+  const formatEventTypeName = (str) => str.toLowerCase().replaceAll("_", " ");
```

Comment:
This simple one-liner  can probably just be used in place in the template.  Or it can be imported. This isn't really the right place for it. Passing it in as a prop is odd.